000 - glossary.md

```diff
@@ -1,39 +1,56 @@
 # Glossary of Terms
 
 This page is intended to be a quick reference to terms that we use throughout
-this guide. These terms are defined within the context of cloud-native software
-systems – more general definitions also exist for more general contexts but are
-not provided here.
+this project.
 
 ---
 
 _Distributed System_: A system that's characterized by its components
 being split up across multiple distinct computers on a network.
 
+_Distributed Tracing_: A method of profiling and monitoring distributed system
+performance by capturing telemetry data associated with each request.
+
 _Instrument_: A logical object that emits a specific piece of named telemetry.
 For example, an instrument might measure the current number of requests being
 handled by a process, or the amount of free disk space.
 
 _Instrumentation_: Code that exists to emit telemetry signals to some external
 system or endpoint. For example, a logging statement or a metric instrument.
 
+_Logs_: Human-readable text output from services and computers in a system.
```

Comment:
I think of Logs as  being both structured and unstructured (the "superclass", as it were)... i.e., they aren't necessarily human-readable text output.